src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,42 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard/MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface Card {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+}
```

Comment:
why not to create and export the file card.ts with this interface (it will prevent code doubling)